api/routes.go

```diff
@@ -77,7 +77,7 @@ func (s *server) routes() {
 	api.HandleFunc("/{account}/volumes/{id}/tags", s.VolumeUpdateHandler).Methods(http.MethodPut)
 
 	api.HandleFunc("/{account}/instances/{id}", s.InstanceDeleteHandler).Methods(http.MethodDelete)
-	api.HandleFunc("/{account}/instances/{id}/volumes/{vid}", s.ProxyRequestHandler).Methods(http.MethodDelete)
+	api.HandleFunc("/{account}/instances/{id}/volumes/{vid}", s.VolumeDetachHandler).Methods(http.MethodDelete).Queries("force", "{force}")
```

Comment:
I think using `Queries` here makes `force` a required parameter, but I'm not sure if we always pass it. You may want to test from the UI

Reply:
I have made the `FORCE` query parameter optional in the code. Please review.